cdc/log.cc

```diff
@@ -337,7 +376,7 @@ cdc::options::options(const std::map<sstring, sstring>& map) {
         }
     }
 
-    if (_enabled && !_preimage && !_postimage && _delta_mode == delta_mode::off) {
+    if (_enabled && !preimage() && !postimage() && get_delta_mode() == delta_mode::off) {
```

Comment:
Again, for consistency, maybe `get_delta_mode()` should or can be called `delta()`.
Of course, doesn't need to be fixed in this patch.

Reply:
All for that. I did not name the accessor, so did not change it here.